docs/docs/concepts/server-mcp.md

```diff
@@ -223,14 +223,19 @@ def get_mcp_tools_node(state, config):
         }
     })
     tools = await client.get_tools() # (3)
-    return {"tools": tools}
-	
+    
+    # Your tool-calling logic here
+    
+    tool_messages = ...
+    return {"messages": tool_messages}
 ```
 
 1. MCP only supports adding headers to requests made to `streamable_http` and `sse` `transport` servers.
 2. Your MCP server URL.
 3. Get available tools from your MCP server.
 
+_This can also be done by [rebuilding your graph at runtime](https://langchain-ai.github.io/langgraph/cloud/deployment/graph_rebuild/) to have a different configuration for a new run_
```

Comment:
@eyurtsev wdyt of having this shown as the default way? then could put in a TOolNode and go